dev/com.ibm.ws.jpa_22_fat/test-applications/jpa22injection/src/jpa22injection/web/JPAInjectionTestServlet.java

```diff
@@ -0,0 +1,161 @@
+package jpa22injection.web;
+
+import javax.annotation.Resource;
+import javax.naming.InitialContext;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.PersistenceContext;
+import javax.persistence.PersistenceUnit;
+import javax.servlet.annotation.WebServlet;
+import javax.transaction.UserTransaction;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import jpa22injection.entity.InjectEntityA;
+import jpa22injection.entity.InjectEntityB;
+import junit.framework.Assert;
+
+@SuppressWarnings("serial")
+@WebServlet(urlPatterns = "/TestJPA22Injection")
+@PersistenceContext(name = "em1", unitName = "JPAPU-1")
+@PersistenceContext(name = "em2", unitName = "JPAPU-2")
+@PersistenceUnit(name = "em1RL", unitName = "JPAPU-1RL")
+@PersistenceUnit(name = "em2RL", unitName = "JPAPU-2RL")
+public class JPAInjectionTestServlet extends FATServlet {
+    @Resource
+    private UserTransaction tx;
+
+    /**
+     * Verify that the @PersistenceContext annotation is repeatable, and that consumable
+     * EntityManagers are provided in the scenario that this capability is utilized in.
+     * 
+     */
+    @Test
+    @Mode(TestMode.LITE)
+    public void testInjectPersistenceContext() throws Exception {
+        final String testName = "testInjectPersistenceContext";
+        System.out.println("STARTING " + testName + " ...");
+
+        try {
+            InitialContext ic = new InitialContext();
+            Assert.assertNotNull(ic);
```

Comment:
calling `new InitialContext()` will never be null.  Overall, the JNDI lookups could be simplified by changing to something like:
`EntityManager em1 = InitialContext.doLookup("java:comp/env/em1");`

Reply:
Updated